src/util/apiHelpers.js

```diff
@@ -75,12 +75,10 @@ const possibleTypes = {
 
 const typePolicies = {
   SearchContext: {
-    keyFields: object =>
-      object.filters?.length ? object.filters.id : object.path,
+    keyFields: object => object.path,
```

Comment:
```suggestion
    keyFields: ['path'],
```
Disse kan forenkles som for FrontpageSearchResult nå